ESSArch_Core/search/importers/tests/test_earderms.py

```diff
@@ -0,0 +1,1254 @@
+                    <Egenskap Namn='Adress'><Varde>some post address</Varde></Egenskap>
+                    <Egenskap Namn='Postnummer'><Varde>some zipcode</Varde></Egenskap>
+                </subroot>
+            </root>
+        '''
+
+        expected_data = {
+            'name': 'some name',
+            'address': 'some post address',
+            'zipcode': 'some zipcode',
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_initiator(root)
+
+        self.assertDictEqual(data, expected_data)
+
+
+class ParseRestrictionTests(TestCase):
+
+    def setUp(self):
+        self.importer = EardErmsImporter()
+
+    def test_parse_restriction_without_AnnanTyp_attr_should_get_Typ_and_all_texts(self):
+        xml_string = '''
+            <root Typ='some type'>
+                <ForklarandeText>some desc text</ForklarandeText>
+                <Lagrum>some lagrum</Lagrum>
+                <RestriktionsDatum>some expiration date</RestriktionsDatum>
+            </root>
+        '''
+
+        expected_data = {
+            'beskrivning': 'some desc text',
+            'lagrum': 'some lagrum',
+            'upphor': 'some expiration date',
+            'typ': 'some type'
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_restriction(root)
+
+        self.assertDictEqual(data, expected_data)
+
+    def test_parse_restriction_without_some_keys_should_catch_IndexError_and_continue(self):
+        xml_string = '''
+            <root Typ='some type'>
+                <ForklarandeText>some desc text</ForklarandeText>
+                <Lagrum>some lagrum</Lagrum>
+            </root>
+        '''
+
+        expected_data = {
+            'beskrivning': 'some desc text',
+            'lagrum': 'some lagrum',
+            'typ': 'some type'
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_restriction(root)
+
+        self.assertDictEqual(data, expected_data)
+
+    def test_parse_restriction_with_AnnanTyp_attr_should_get_AnnanTyp_and_all_texts(self):
+        xml_string = '''
+            <root Typ='some type' AnnanTyp='some other type'>
+                <ForklarandeText>some desc text</ForklarandeText>
+                <Lagrum>some lagrum</Lagrum>
+                <RestriktionsDatum>some expiration date</RestriktionsDatum>
+            </root>
+        '''
+
+        expected_data = {
+            'beskrivning': 'some desc text',
+            'lagrum': 'some lagrum',
+            'upphor': 'some expiration date',
+            'typ': 'some other type'
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_restriction(root)
+
+        self.assertDictEqual(data, expected_data)
+
+
+class ParseGallringTests(TestCase):
+
+    def setUp(self):
+        self.importer = EardErmsImporter()
+
+    def test_parse_gallring_when_gallars_is_true(self):
```

Comment:
```suggestion
    def test_parse_gallring_when_gallras_is_true(self):
```

Reply:
You eagle eyed man! How do you spot my secret typos!? :)